src/runtime_config/actions.js

```diff
@@ -68,6 +68,30 @@ function checkLength(value, min, max, field) {
   }
 }
 
+function getUniqueKvKey() {
+  // When a KV map only contains one value, this is the key at which
+  // the value is recorded
+  return new ArrayBuffer(8);
+}
+
+function getActiveRecoveryMembersCount() {
+  let activeRecoveryMembersCount = 0;
+  ccf.kv["public:ccf.gov.members.encryption_public_keys"].forEach((_, k) => {
+    let member_info = ccf.kv["public:ccf.gov.members.info"].get(k);
+    if (member_info === undefined) {
+      throw new Error(
+        `Recovery member ${ccf.bufToJsonCompatible(k)} has no information`
```

Comment:
Why not bufToStr(), isn't this a member_id?

Reply:
Good spot!